### ISSUE TITLE
STEM support

### ISSUE THREAD

**Issue by eric-schneider** [riptano/docs-ui#90]
There are [certain pages](https://docs.datastax.com/en/astra/astra-db-vector/api-reference/data-api-commands.html#cosine-metric) in the docs where we may need to display equations and other types of math. Currently, we have to manually create SVG files to present this kind of information, which is labor-intensive to create, update, and make compatible with our CSS styles.

To alleviate this problem, the AsciiDoc language provides [STEM support](https://docs.asciidoctor.org/asciidoc/latest/stem/) for embedding math-mode macros from [LaTeX](https://docs.mathjax.org/en/latest/input/tex/index.html) and/or [AsciiMath](https://docs.mathjax.org/en/latest/input/asciimath.html) notation as block or inline elements. These elements act as passthroughs to preserve the expressions as entered. The expressions are then passed on to the converter to be processed and rendered for display using a STEM provider (e.g., MathJax).

The STEM provider must be configured in the output converter to render the expressions. As far as implementation goes, we should thoroughly review the discussion in [Antora Issue #184](https://gitlab.com/antora/antora/-/issues/184) and investigate the options presented. The simplest approach would probably be to load MathJax via CDN on just the pages that use `stem`. (That's the approach the Apache Cassandra project went with: https://github.com/apache/cassandra-website/pull/215.) However, as pointed out in the Antora discussion, using a CDN is a security risk. But providing an offline alternative might be tricky. The offline approach presented in [this comment](https://gitlab.com/antora/antora/-/issues/184#note_291908945) might be a good place to start since it appears to have the semi-endorsement of the Antora maintainers.
